s2_jv_electric_boogaloo/main.cpp

```diff
@@ -0,0 +1,9 @@
+#include <QApplication>
+
+int main(int argc, char* argv[])
+{
+    QApplication a(argc, argv);
+    test1 w;
```

Comment:
On devrait remplacer `test1` par un nom plus significatif, j'suis pas certain de comprendre c'est quoi l'object test1

Reply:
En fait, est-ce que c'est juste un fichier qui a été commit par erreur? Notre vrai main semble être dans  `s2_jv_electric_boogaloo.cpp`. Si oui, delete this 🔫